dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -183,7 +183,6 @@ metrics-jmx/4.1.1//metrics-jmx-4.1.1.jar
 metrics-json/4.1.1//metrics-json-4.1.1.jar
 metrics-jvm/4.1.1//metrics-jvm-4.1.1.jar
 minlog/1.3.0//minlog-1.3.0.jar
-mssql-jdbc/6.2.1.jre7//mssql-jdbc-6.2.1.jre7.jar
```

Comment:
FYI, you need to find the root cause from the pom file and remove it.

Reply:
LOL, out of https://github.com/apache/spark/pull/28635 it's not removed :)